### PR TITLE
Improve logs performances when coming back and forth

### PR DESCRIPTION
**Pull request by Warry**
- Filters out debug, when not activated
- Caches the dom

### CODE REVIEW

File: ui/app/assets/services/sbt/logs.js [hunk 1]
```diff
@@ -29,15 +29,24 @@ define([
...
+      return m.event.entry.level !== "debug" || (app.settings.showLogDebug() || debug);
+    else
+      return true;
+  }
+
   /**
   Logs, by execution/task
   */
   logEvent.matchOnAttribute("subType", "TaskLogEvent")
+    .filter(filterDebug)
```
Comment (havocp) [inline ui/app/assets/services/sbt/logs.js]: is the debug filtering still necessary with the DOM cached?

**Comment by havocp** [comment]
This fix looks like the right solution to me. It's a focused change that addresses the exact problem (dom recreation).

**Comment by henrikengstrom** [comment]
LGTM. Do we want to still use filtering of debug though? I tried it without any filtering and it seems pretty fast to me without the filtering too.

File: ui/app/assets/services/sbt/logs.js (hunk 1 above)
Comment (Warry) [inline ui/app/assets/services/sbt/logs.js]: It mechanically increases the scrollback, while the debug functionality is really supposed to be off 99% of the time.

**Comment by henrikengstrom** [comment]
Okay, merging this one now and we can amend the filtering in case we want to later.

**Pull request merged**